src/client/client.js

```diff
@@ -188,8 +199,15 @@ function Client({ game, numPlayers, board, multiplayer, debug }) {
       );
     }
 
+    updateState() {
+      this.setState({
+        store: this.store.getState(),
```

Comment:
Given that you're putting this in `store`, isn't it going to break line 166, i.e. `G` and `ctx` are no longer going to be at the top level of the passed in props.

Reply:
I have to double check, but my understanding is that we are not reading this component's state anywhere. The state we pass on line 166 is from redux store, not this component's state. I had to add the new parameters to the state because if I did not, React would not re-render the component after the connection dropped (for instance).

All the examples were running correctly, but just to be sure, I will double check and manually test with the debug UI activated again to see that it does not break.

Reply:
So yes, I confirmed that this should not be an issue. The state we are updating here is Client's state, not the state we pass to the child component through the redux connect() call. This should not affect the props passed to it. 

We are also safe because G is checked in the unit test: See line 130 of src/client/client.test.js